scripts/test-e2e-kubernetes.sh

```diff
@@ -121,8 +127,8 @@ fi
 if [[ -z "${SKIP_CONTAINER_IMAGE_PUBLISHING:-}" ]]; then
   # Make the container image accessible to the k8s cluster.
   if is_minikube_cache_enabled; then
-    minikube cache add "$CONTAINER_IMAGE"
-    trap 'minikube cache delete "$CONTAINER_IMAGE"' EXIT
+    minikube image load "$CONTAINER_IMAGE"
+    trap 'minikube image rm "$CONTAINER_IMAGE"' EXIT
```

Comment:
`minikube image` is the new syntax right? We'll need to make sure the existing version used in CI supports it, or (preferably in my opinion) upgrade the minikube used in CI

Reply:
I just confirmed our minikube version in CI _doesn't_ have the `image` subcommand

Reply:
Indeed, I push a naive upgrade attempt, let see if CI is happy with it

Reply:
The CI is all green, I don't thing there is any reason not to upgrade minikube.